src/pages/getting-started.js

```diff
@@ -125,17 +125,18 @@ const IndexPage = () => {
             <main className="container content-main-container content documentation-page">
                 <h1 id="getting-started" className={'main-title'}>Getting started</h1>
 
-                <p>Nickel is still young, and the installation process is not yet optimal. Sorry about that! We are focused on improving the
-                    experience, so stay tuned. </p>
+                The simplest way to get nickel and nls (the Nickel language server) binaries is to download them from the pre-built binaries available
+                under the Assets section of the latest <a className={"link-primary"} href="https://github.com/tweag/nickel/releases">GitHub release</a>.
+
+                If your platform isn't available, or if you prefer to use a package manager, we give three alternative ways of obtaining a running Nickel executable:
 
-                We give three alternative ways of obtaining a running Nickel executable:
                 <ol>
                     <li>Using <a className={"link-primary"} href="https://nixos.org/">Nix</a></li>
                     <li>Using <a className={"link-primary"} href="https://doc.rust-lang.org/cargo/">Cargo</a></li>
                     <li>Using <a className={"link-primary"} href="https://www.docker.com/">Docker</a></li>
                 </ol>
 
-                If you have neither of these tools, Nix is the preferred way. If you don't have Nix but you are a user of Cargo or Docker, you may prefer the corresponding installation method.
+                If you have neither of these tools, Nix is the preferred way. If you don't have Nix but you are Cargo or Docker user, you may prefer the corresponding installation method.
```

Comment:
Maybe we don't need this sentence anymore? The prebuilt binaries are the preferred way, and you'd only want one of these 3 if you already use the corresponding tool

Reply:
Your architecture could also be unsupported (with respect to pre-built binaries). Old x86 MacOS for example. Another point is if you want to install Nickel through a package manager, for updates and such. Maybe the "preferred way" is confusing though, as if Nix is the preferred way of all install methods?

Another question is that if it's the right call. Nix is much more invasive that cargo on a fresh install. Maybe cargo should actually be the preferred, minimalist way of installing from source.